pkg/controller/windowsmachineconfig/windows/windows.go

```diff
@@ -90,6 +103,94 @@ func (vm *Windows) initializeBootstrapperFiles() error {
 	return nil
 }
 
+// handleHybridOverlay ensures that the hybrid overlay is running on the node
+func (vm *Windows) handleHybridOverlay(nodeName string) error {
+	// Check if the hybrid-overlay is running
+	_, stderr, err := vm.Run("Get-Process -Name \""+hybridOverlayProcess+"\"", true)
+
+	// stderr being empty implies that hybrid-overlay was running.
+	if err == nil || stderr == "" {
+		// Stop the hybrid-overlay
+		stopCmd := "Stop-Process -Name \"" + hybridOverlayProcess + "\""
+		_, stderr, err := vm.Run(stopCmd, true)
+		if err != nil || stderr != "" {
+			log.Info("%s:\n%s", stopCmd, stderr)
+			return errors.Wrap(err, "unable to stop hybrid-overlay")
+		}
+	}
+
+	_, stderr, err = vm.Run(mkdirCmd(logDir), false)
+	if err != nil {
+		return errors.Wrap(err, fmt.Sprintf("unable to create %s directory:\n%s", logDir, stderr))
+	}
+
+	if err := vm.CopyFile(wkl.HybridOverlayPath, remoteDir); err != nil {
+		return errors.Wrap(err, fmt.Sprintf("error copying %s-->%s", wkl.HybridOverlayPath,
+			remoteDir+wkl.HybridOverlayName))
+	}
+
+	// Start the hybrid-overlay in the background over ssh. We cannot use vm.Run() and by extension WinRM.Run() here as
```

Comment:
Can you add a TODO here with the link for the story which fixes this: https://issues.redhat.com/browse/WINC-296